staging/src/k8s.io/component-helpers/node/util/status.go

```diff
@@ -38,7 +38,7 @@ func PatchNodeStatus(c v1core.CoreV1Interface, nodeName types.NodeName, oldNode
 
 	updatedNode, err := c.Nodes().Patch(context.TODO(), string(nodeName), types.StrategicMergePatchType, patchBytes, metav1.PatchOptions{}, "status")
 	if err != nil {
-		return nil, nil, fmt.Errorf("failed to patch status %q for node %q: %v", patchBytes, nodeName, err)
+		return nil, nil, fmt.Errorf("failed to patch status %q for node %q: %w", patchBytes, nodeName, err)
```

Comment:
nit: looks unrelated. Maybe a separate PR (fixing other similar cases in the kubelet code)?

Reply:
It's related. `apierrors.IsConflict()` needs the error to be wrapped (i.e. "%w"). I also filed https://github.com/kubernetes/kubernetes/issues/123234 as I think this is a more general issue for whole code base. The conclusion in that issue seems to suggest only change it when the intent is clearly to wrap the error, which is the case here.